plugins/modules/win_get_url.ps1

```diff
@@ -201,6 +219,9 @@ Function Invoke-DownloadFile {
 
         if ($download) {
             Copy-Item -LiteralPath $tmp_dest -Destination $Dest -Force -WhatIf:$Module.CheckMode > $null
+            if ($SetZone) {
```

Comment:
We probably want to skip if in `$Module.CheckMode` as the file may not actually exist and we don't want to make a change to it anyway in check mode.